Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Nit: For developer convenience, can we rename this away from starting with Build? I really like tab-completing B<tab>uildGVFSForMac.sh automatically.
Maybe CreateInstaller.sh?

Reply:
Agreed on avoiding that collision, though this script should really live in `src/GVFS/GVFS.Installer.Mac`, not here in the Scripts folder. This is a source file that produces a build output, not a helper script.

Reply:
Renamed the script to CreateMacInstaller.sh and moved it to inside of GVFS.Installer directory.

<img width="615" alt="screen shot 2018-11-07 at 12 35 03 pm" src="https://user-images.githubusercontent.com/378580/48149285-b4695900-e289-11e8-8cf3-081b452d1481.png">